.vib/grafana/cypress/cypress/support/commands.js

```diff
@@ -12,6 +12,16 @@ for (const command of ['click']) {
   });
 }
 
+Cypress.on('uncaught:exception', (err, runnable) => {
+  // In Grafana 9.3.0 the console throws a typeError with message
+  //'Cannot read properties of undefined (reading 'eventTrackingNamespace')'.
+  // An issue has been created:
+  // https://github.com/grafana/grafana/issues/60377
+  if (err.message.includes("Cannot read properties of undefined (reading 'eventTrackingNamespace')")) {
```

Comment:
Should the error message be so specific or would a simple **"Cannot read properties of undefined"** be enough?

Reply:
I'd rather use a specific message in this case, to not cover up any other errors.